FindmallocMC.cmake

```diff
@@ -96,8 +93,14 @@ if(mallocMC_ROOT_DIR)
     string(REGEX MATCH "([0-9]+)" mallocMC_VERSION_PATCH
                                 ${mallocMC_VERSION_PATCH_HPP})
 
+    # mallocMC variables ########################################################
+    #
+    set(mallocMC_REQUIRED_VARS_LIST mallocMC_ROOT_DIR ; mallocMC_INCLUDE_DIRS)
```

Comment:
this is actually a list of "var1" ";" and "var2": [ref](http://www.cmake.org/Wiki/CMake:VariablesListsStrings#Strings_vs._lists) - pls change